### PR TITLE
Fix the issue where the backend API URL was not configurable. 

### LINKED ISSUE #69: Setting Up Backend Server URL in Local Environment and Frontend Environment
**Describe the bug**
Setting Up Backend Server URL in Local Environment and Frontend Environment. 
**To Reproduce**
1. Open the `constants.ts` file.
2. Notice that the `API_URL` constant and see API URL 

**Expected behavior**
The `API_URL` should be set to the URL of the backend server.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, macOS, Linux]
 - IDE: [e.g. VS Code, IntelliJ]
 - Version: [e.g. 1.60]

**Additional context**
This issue is critical for new contributors who are setting up the project for the first time. They might not know that they need to set the `API_URL`.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by mayura-andrew**
## Purpose
The purpose of this PR is to fix #69 
## Goals
This PR makes the backend API URL configurable through the `VITE_API_URL` environment variable. This allows the application to connect to different backend APIs depending on the environment.

## Approach
I updated the `README.md` file to include instructions on how to set the `VITE_API_URL` environment variable. See the changes in this PR for more details.

## Screenshots
N/A

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
N/A

## Test environment
OS - Fedora Linux 39 (KDE Plasma) x86_64
NodeJS - 20.10.0

## Learning
I learned about the importance of making backend API URLs configurable to support different environments. I also learned about Vite's support for loading environment variables.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+VITE_API_URL=http://localhost:4000/api
```
Comment (YoshithaRathnayake) [inline .env]: Can you add the end of the line @mayura-andrew 
```suggestion
VITE_API_URL=http://localhost:4000/api

```

File: example.env [hunk 2]
```diff
@@ -1 +1 @@
-# This is an example environment file for configuring dynamic values.
+VITE_API_URL=http://localhost:4000/api
```
Comment (YoshithaRathnayake) [inline example.env]: here also 
```suggestion
VITE_API_URL=http://localhost:4000/api

```

**Review by YoshithaRathnayake — DISMISSED** [review]

File: example.env (hunk 2 above)
Comment (mayura-andrew) [reply example.env]: okay sure i''ll do thank you for mentioning this i forgot to add this

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Instead of the current approach, I suggest the following alternative:

1. Generate a file named `example.env`.
2. Include the necessary environment variables in this file.
3. Instruct the developer to duplicate this file and modify the environment variables accordingly.

To achieve this, use the following command and ensure it is documented in the README file:

```bash
cp example.env .env
```

Make sure to include this information in the README file for reference.

**Comment by mayura-andrew** [comment]
Got it, previously I implemented this approach and removed. I'll update this

File: .env (hunk 1 above)
Comment (anjula-sack) [inline .env]: Remove this .env file

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (mayura-andrew) [reply .env]: Done 👍🏻

**Review by anjula-sack — APPROVED** [review]
Thank you! @mayura-andrew

**Pull request merged**